src/main/java/com/magnaboy/CitizensOverlay.java

```diff
@@ -81,13 +81,39 @@ private void highlightTile(Graphics2D graphics, WorldPoint wp, Color color) {
 		}
 	}
 
+	private void highlightRegion(Graphics2D graphics, WorldPoint bottomLeft, WorldPoint topRight, Color color)
+	{
+		WorldArea boundingBox = Util.calculateBoundingBox(bottomLeft, topRight);
+		highlightRegion(graphics, boundingBox, color);
+	}
+
+	private void highlightRegion(Graphics2D graphics, WorldArea boundingBox, Color color)
+	{
+		int x = boundingBox.getX();
+		int y = boundingBox.getY();
+		for (int i = y; i <= y + boundingBox.getHeight(); i++) {
+			for (int t = 0; t <= boundingBox.getWidth(); t++) {
+				highlightTile(graphics, new WorldPoint(x + t, i, 0), color);
```

Comment:
We cannot ever assume the plane is 0. If we add citizens on other planes, this will not work.

Reply:
Fixed